com.unity.ml-agents/Runtime/SensorHelper.cs

```diff
@@ -0,0 +1,61 @@
+using UnityEngine;
+
+namespace Unity.MLAgents.Sensors
+{
+    public static class SensorHelper
```

Comment:
Moved this out of VectorSensorTests.cs so that we can reuse the logic in the extension tests.

I tried exposing the `Unity.MLAgents.Tests` version, and got it working locally but couldn't get it compiling on yamato. asmdef experts welcome to try to improve.